config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     - name: ANSIBLE_ENABLE_TASK_DEBUGGER
       value: "True"
   services:
+    - repo-setup
```

Comment:
repo-setup should not be in the sample here. it's added by install_yamls.

Reply:
This PR is passing now. So, I think we just need to remove this and then we're good here.

Reply:
It was added in https://github.com/openstack-k8s-operators/dataplane-operator/commit/8bdfcb1bf948a126c128fee86d8bd34af81aaaa2a and then removed in a later patch it seems.